File: 프로그래머스/1/86491. 최소직사각형/최소직사각형.swift
import Foundation

func solution(_ sizes:[[Int]]) -> Int {
    var sizes = sizes
    
    var x: [Int] = []
    var y: [Int] = []
    
    for i in 0..<sizes.count {
        sizes[i].sort()
        
        x.append(sizes[i][0])
        y.append(sizes[i][1])
    
    }
    return x.max()! * y.max()!
}

File: 프로그래머스/1/86491. 최소직사각형/최소직사각형.py
def solution(sizes):
    answer = 0
    width, height = 0, 0
    
    for w,h in sizes:
        w, h = min(w,h), max(w,h)
        width = max(width, h)
        height = max(height, w)
        
    return width * height